gogs/README.md

```diff
@@ -0,0 +1,124 @@
+[![HitCount](http://hits.dwyl.com/dwyl/learn-devops-gogs.svg)](http://hits.dwyl.com/dwyl/learn-devops-gogs)
+
+</div>
+
+# _Why_? 🤷‍♂️
+
+You need a simple **Git server** 
+without the steep learning curve of 
+[Git SCM](https://git-scm.com/book/en/v2/Git-on-the-Server-Setting-Up-the-Server)
+or _overhead_ of running a **GitLab** instance.
+
+**`Gogs`** is a performance-friendly 
+Git web interface written in the **`Go`** programming language. 
+It claims to make setting up and managing GIT painless;
+let's test that!
+
+
+# _What_? 📦
+
+In our case **`@dwyl`**,
+we are _mostly_ happy with the GitHub service. 👌<br />
+Except for a few annoying aspects of the interface
+and the occasional outage, GitHub is good.
+We are exploring having a **`Gogs`** server running as a 
+[**_hot_ backup**](https://en.wikipedia.org/wiki/Backup_site#Hot_site)
+which we can use when GitHub is unavailable
+or if they ever lose our data! 
+
+# _Who_? 🤓
+
+The audience for this guide is "_us_" 
+[**`@dwyl`**](https://github.com/dwyl). <br />
+If anyone `else` finds it helpful that's a bonus. <br />
+Please ⭐ the repo if you find it useful. Thanks!
+
+# _How_? 👩‍💻
+
+There are several options for running `Gogs`:
+1. **Directly** on your main machine/workstation -
+    can be advantageous in terms of setup speed,
+    but clutters your main machine with a server
+    that requires resources + maintenance.
+
+2. On your machine using **Docker** - provides
+   good isolation/separation, 
+   reasonably fast startup times
+   and can be easily "killed" when no longer needed.
+
+3. On a **_secondary_ computer** e.g. **RaspberryPi** - complete isolation,
+   more "hackable" and simulates production more closely.
+   Obviously having a second machine is a luxury 
+   not everyone has. But we figure a 
+   [$15](https://www.raspberrypi.com/products/raspberry-pi-zero-2-w/)
+   RasbperryPi Zero is not a stretch for most people doing DevOps.
+
+4. On **3rd Party/Cloud Service** e.g. AWS or **Fly.io** -
+   perfect once you have decided `Gogs` is for you 
+   and you want a Production instance for your project.
+   We use Fly.io because it's: <br />
+   a) Simplicity: deploying to Fly.io is 
+   _much_ simpler than other VPS/Cloud services (AWS!). <br />
+   b) _Cheaper_ than an equivalent ($2/month) <br />
+   c) Our _main_ app already runs on Fly.io 
+   so having the `Gogs/Git` server co-located 
+   is good for latency/speed/performance.
+
+We chose to install `Gogs` on a **RaspberryPi**
+for 3 reasons: <br />
+a) Keeps it fully isolated on a separate machine. 
+Which simulates "production". <br />
+b) Allows us to test connecting to it over a network. i.e. real world tests.<br />
+c) Means we can run the instance long-term 
+and store actual (backup) data on it.<br />
+
+There are quite a few steps for setting this up,
+so we documented them in a separate doc.
+See: [`/install-gogs-raspberrypi.md`]()
```

Comment:
Do we need to specify the link:
```suggestion
See: [`/install-gogs-raspberrypi.md`](./install-gogs-raspberrypi.md)
```
Or is Github clever enough to created from the name?

Reply:
nope. GitHub is definitely not that smart. a link without an `src` just goes to the current page. I hadn't updated these cause I didn't have the full URLs yet. but let's add this and see if it works once on the `main` branch. 💭